novaboot

```diff
@@ -1150,9 +1173,23 @@ host server {
 
 if (defined $dhcp_tftp || defined $tftp) {
     $tftp_port ||= 69;
+    my $tftp_root = "$builddir";
+    if (defined $server) {
+        $tftp_root = $tftp_root . "/$server";
```

Comment:
What if server is an absolute path?

Reply:
yep good catch, changed to just `$server`